enterprise/templates/enterprise/enterprise_course_enrollment_page.html

```diff
@@ -88,6 +90,64 @@ <h3 class="course-confirmation-title">${ confirmation_text }</h3>
         </div>
       </div>
     </div>
-
+    <div id="course-details-modal" class="modal" aria-hidden="true">
+      <div id="course-details-modal-content" class="modal-content" role="dialog" aria-modal="true" aria-labelledby="modal-header-text">
+        <button class="modal-close-button" id="modal-close-button" aria-label="${_('Close')}">${_('Close')}</button>
+        <header class="wrapper">
+          <div class="image">
+            <img src="${ course_image_uri }" alt="${ course_name }"/>
+          </div>
+          <div class="details">
+            <h1 id="modal-header-text">${ course_name }</h1>
+            <p class="short-description">${ course_short_description }</p>
+            <div class="enterprise">
+              <img src="${ enterprise_customer.branding_configuration.logo.url }" alt="${ enterprise_customer.name }" />
+            </div>
+          </div>
+        </header>
+        <div class="body">
+          <div class="details">
+            <ol>
+              <li>
+                <span class="icon fa fa-tag" aria-hidden="true"></span>
+                <span class="title">${ price_text}</span>
+                <span class="text">
+                  % if course_modes:
+                    % if course_modes[0]['final_price'] and course_modes[0]['original_price'] != course_modes[0]['final_price']:
+                    <strike>${ course_modes[0]['original_price'] }</strike>
+                    <br />
+                    <span class="discount">${ course_modes[0]['final_price'] }</span>
+                    % else:
+                    ${ course_modes[0]['original_price'] }
+                    % endif
+                  % endif
+                </span>
+              </li>
+              <li>
+                <span class="icon fa fa-certificate" aria-hidden="true"></span>
+                <span class="title">${ level_text}</span>
+                <span class="text">Intermediate</span>
+              </li>
+              <li>
+                <span class="icon fa fa-tachometer" aria-hidden="true"></span>
+                <span class="title">${ effort_text}</span>
+                <span class="text">${ course_effort }</span>
+              </li>
+              <li>
+                <span class="icon fa fa-clock-o" aria-hidden="true"></span>
+                <span class="title">${ duration_text}</span>
+                <span class="text">12 weeks per course</span>
+              </li>
+            </ol>
+          </div>
+          <div class="overview">
+            <h1><strong>What you'll mock</strong></h1>
+            <p>Is this what Studio calls "Course Overview"? Should this be inserted as an HTML fragment? What is the name of the attribute to look and where</p>
```

Comment:
Yes, this should be an HTML fragment. I would just have a div here that can be filled into; you should be able to get this from `course_details['overview']`.